src/main/java/com/hivemq/extension/sdk/api/client/parameter/ConnectionInformation.java

```diff
@@ -74,9 +74,21 @@
 
     /**
      * Information about TLS, should the client be connected to HiveMQ via an TLS listener.
+     * This information is only present if the client connected with mutual TLS.
      *
      * @return An {@link Optional} of the {@link TlsInformation} of the client.
      * @since 4.0.0, CE 2019.1
+     * @deprecated Use {@link #getClientTlsInformation()} instead.
      */
+    @Deprecated
     @NotNull Optional<TlsInformation> getTlsInformation();
+
+    /**
+     * Information about TLS, should the client be connected to HiveMQ via an TLS listener.
+     * This information is only present if the client connected with mutual TLS.
+     *
+     * @return An {@link Optional} of the {@link ClientTlsInformation} of the client.
+     * @since 4.5.2, CE 2021.1
```

Comment:
Does this `@since` annotation clash with the `ClientTlsInformation` `@since` annotation?